Gems/Profiler/Code/Source/ImGuiCpuProfiler.cpp

```diff
@@ -462,9 +475,10 @@ namespace Profiler
             m_groupRegionMap[*groupNameItr][*regionNameItr].RecordRegion(newRegion, entry.m_threadId);
         }
 
-        // Update viewport bounds with some added UX fudge factor
-        m_viewportStartTick = deserializedData.back().m_startTick - 1000;
-        m_viewportEndTick = deserializedData.back().m_endTick + 1000;
+        // Update viewport bounds to the estimated final frame time with some padding
+        const AZStd::sys_time_t padding = 5000;
```

Comment:
Nitpick: magic number can be defined globally as `constexpr` and with a more descriptive name like "ProfilerViewEdgePadding".